### PR TITLE
Adjust class hierarchy for caudal ganglionic eminence derived interneuron

### LINKED ISSUE #2357: Adjust Class hierarchy add synonyms to caudal ganglionic eminence derived interneuron
I think this term [caudal ganglionic eminence derived GABAergic cortical interneuron](http://purl.obolibrary.org/obo/CL_4023070) should just be a synonym of [caudal ganglionic eminence derived interneuron](http://purl.obolibrary.org/obo/CL_4023064) and not a different term. 

I had a look at a few papers, and they do not really describe CGE-derived interneurons that are not GABAergic. I can look a bit further if a discussion is needed.

**cell label:**
[caudal ganglionic eminence derived interneuron](http://purl.obolibrary.org/obo/CL_4023064)

**parent terms**
[interneuron](http://purl.obolibrary.org/obo/CL_0000099)

**downstream terms:**
[vip GABAergic cortical interneuron](http://purl.obolibrary.org/obo/CL_4023016)
[lamp5 GABAergic cortical interneuron](http://purl.obolibrary.org/obo/CL_4023011)
Lamp5 Lhx6 [NTR - #2345 ]
PAX6 neuron [NTR]
[sncg GABAergic cortical interneuron](http://purl.obolibrary.org/obo/CL_4023015)
[vip GABAergic cortical interneuron](http://purl.obolibrary.org/obo/CL_4023016)

**anatomical location**
neocortex

**Text definition (already present):**
A GABAergic cortical interneuron that develops from the caudal ganglionic eminence and has migrated to the cerebral cortex.

**database_cross_reference**
[PMID:20130169](https://pubmed.ncbi.nlm.nih.gov/20130169)
[PMID:37824663](https://pubmed.ncbi.nlm.nih.gov/37824663/)
[PMID:37824655](https://pubmed.ncbi.nlm.nih.gov/37824655/)
[PMID:20130169](https://pubmed.ncbi.nlm.nih.gov/20130169)

**has_a_related_synonym**
CGE-derived cell
CGE-derived GABAergic INs
**has_a_synonym_type**
abbreviation
**database_cross_reference**
[PMID:27034423](https://pubmed.ncbi.nlm.nih.gov/27034423/)

**has_a_related_synonym**
CGE-derived interneuron
**has_a_synonym_type**
abbreviation
**database_cross_reference**
[PMID:20130169](https://pubmed.ncbi.nlm.nih.gov/20130169)
[PMID:37824655](https://pubmed.ncbi.nlm.nih.gov/37824655/)

**has_a_related_synonym**
CGE cortical interneuron
**database_cross_reference**
[PMID:20130169](https://pubmed.ncbi.nlm.nih.gov/20130169)

**has_a_related_synonym**
 [caudal ganglionic eminence derived GABAergic cortical interneuron](http://purl.obolibrary.org/obo/CL_4023070)
**database_cross_reference**
[PMID:20130169](https://pubmed.ncbi.nlm.nih.gov/20130169)

**has_a_related_synonym**
CGE interneuron
**has_a_synonym_type**
abbreviation
**database_cross_reference**
[PMID:37824663](https://pubmed.ncbi.nlm.nih.gov/37824663/)

**symbol**
CGE interneuron

@aleixpuigb @Caroline-99 @JABelfiore to review, once it is good to go, I will turn this into a PR

### PR DESCRIPTION
**Pull request by AvolaAmg**
fixes #2357

### CODE REVIEW

File: src/ontology/cl-edit.owl [hunk 1]
```diff
@@ -2806,7 +2806,6 @@ Declaration(Class(obo:CL_4023066))
 Declaration(Class(obo:CL_4023067))
 Declaration(Class(obo:CL_4023068))
 Declaration(Class(obo:CL_4023069))
-Declaration(Class(obo:CL_4023070))
```
Comment (anitacaron) [inline src/ontology/cl-edit.owl:2809]: The term was accidentally removed 😬

**Review by anitacaron — CHANGES_REQUESTED** [review]

**Comment by anitacaron** [comment]
Please check this doc https://oboacademy.github.io/obook/howto/obsolete-term/

**Comment by AvolaAmg** [comment]
@anitacaron thank you! I will have a look at it to fix this PR!

File: src/ontology/cl-edit.owl (hunk 1 above)
Comment (AvolaAmg) [reply src/ontology/cl-edit.owl]: Sorry, it is now deprecated.

**Comment by AvolaAmg** [comment]
#gogoeditdiff

File: src/ontology/components/cellxgene_subset.owl [hunk 2]
```diff
@@ -703,2747 +703,2752 @@ Declaration(AnnotationProperty(<http://www.geneontology.org/formats/oboInOwl#inS
...
 
-# Class: <http://purl.obolibrary.org/obo/CL_4023070> (<http://purl.obolibrary.org/obo/CL_4023070>)
+# Class: <http://purl.obolibrary.org/obo/CL_4023070> (obsolete caudal ganglionic eminence derived GABAergic cortical interneuron)
 
 AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#inSubset> <http://purl.obolibrary.org/obo/CL_4023070> <http://purl.obolibrary.org/obo/cl#cellxgene_subset>)
+AnnotationAssertion(rdfs:comment <http://purl.obolibrary.org/obo/CL_4023070> "caudal ganglionic eminence derived GABAergic interneuron
+and caudal ganglionic eminence derived interneuron are the same cell type. 
+The two must be merged and one needs to be defined as obsolete. 
+We decided to deprecate the entity with the more specific term and redundant qualities 
+in the label. See comments in issue 2357.")
```
Comment (anitacaron) [inline src/ontology/components/cellxgene_subset.owl:3409]: This subset is auto-generated, so this annotation will be deleted next time. It should move to the cl-edit.owl file.

**Review by anitacaron — CHANGES_REQUESTED** [review]

File: src/ontology/components/cellxgene_subset.owl (hunk 2 above)
Comment (aleixpuigb) [reply src/ontology/components/cellxgene_subset.owl:3409]: So there is no action to take here, right?

**Comment by anitacaron** [comment]
The QC is failing because there's an inferred equivalence, which is forbidden:

<http://purl.obolibrary.org/obo/CL_0010011> equivalent to <http://purl.obolibrary.org/obo/CL_4023064>

> 2024-05-21 15:12:29,356 ERROR org.obolibrary.robot.ReasonOperation - Only equivalent classes that have been asserted are allowed. Inferred equivalencies are forbidden.
> 2024-05-21 15:12:29,357 ERROR org.obolibrary.robot.ReasonOperation - Equivalence: <http://purl.obolibrary.org/obo/CL_0010011> == <http://purl.obolibrary.org/obo/CL_4023064>

**Comment by anitacaron** [comment]
#gogoeditdiff

**Comment by anitacaron** [comment]
Can't update the diff for the same reason the QC is failing.

**Comment by aleixpuigb** [comment]
#gogoeditdiff

File: src/ontology/cl-edit.owl [hunk 3]
```diff
@@ -29735,11 +29738,19 @@ EquivalentClasses(obo:CL_4023063 ObjectIntersectionOf(obo:CL_0000099 ObjectSomeV
...
 
 AnnotationAssertion(obo:IAO_0000028 obo:CL_4023064 "CGE interneuron")
-AnnotationAssertion(Annotation(oboInOwl:hasDbXref "DOI:10.1101/2022.10.12.511898") obo:IAO_0000115 obo:CL_4023064 "An interneuron that is derived from the caudal ganglionic eminence.")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:20130169") Annotation(oboInOwl:hasDbXref "PMID:37824655") Annotation(oboInOwl:hasDbXref "PMID:37824663") obo:IAO_0000115 obo:CL_4023064 "An interneuron that is derived from the caudal ganglionic eminence.")
 AnnotationAssertion(terms:contributor obo:CL_4023064 <http://orcid.org/0000-0001-7258-9596>)
-AnnotationAssertion(oboInOwl:hasExactSynonym obo:CL_4023064 "CGE interneuron")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:37824663") Annotation(oboInOwl:hasSynonymType obo:OMO_0003000) oboInOwl:hasExactSynonym obo:CL_4023064 "CGE interneuron")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:20130169") oboInOwl:hasExactSynonym obo:CL_4023064 "caudal ganglionic eminence derived GABAergic cortical interneuron")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:20130169") oboInOwl:hasRelatedSynonym obo:CL_4023064 "CGE cortical interneuron")
```
Comment (aleixpuigb) [inline src/ontology/cl-edit.owl:29745]: It's missing the abbreviation property.

File: src/ontology/cl-edit.owl [hunk 4]
```diff
@@ -29166,7 +29166,8 @@ AnnotationAssertion(terms:contributor obo:CL_4023011 <https://orcid.org/0000-000
 AnnotationAssertion(oboInOwl:hasDbXref obo:CL_4023011 "ILX:0770149")
 AnnotationAssertion(oboInOwl:inSubset obo:CL_4023011 cl:BDS_subset)
 AnnotationAssertion(rdfs:label obo:CL_4023011 "lamp5 GABAergic cortical interneuron")
-EquivalentClasses(obo:CL_4023011 ObjectIntersectionOf(obo:CL_0000099 ObjectSomeValuesFrom(obo:RO_0002100 obo:UBERON_0000956) ObjectSomeValuesFrom(obo:RO_0002215 obo:GO_0061534) ObjectSomeValuesFrom(obo:RO_0002292 obo:PR_000032148)))
+EquivalentClasses(obo:CL_4023011 ObjectIntersectionOf(obo:CL_4023064 ObjectSomeValuesFrom(obo:RO_0002100 obo:UBERON_0000956) ObjectSomeValuesFrom(obo:RO_0002215 obo:GO_0061534) ObjectSomeValuesFrom(obo:RO_0002292 obo:PR_000032148)))
```
Comment (aleixpuigb) [inline src/ontology/cl-edit.owl:29169]: Changing the parental terms produces that many cells cannot be classified as this cell type anymore.

File: src/ontology/cl-edit.owl [hunk 5]
```diff
@@ -29211,7 +29212,8 @@ AnnotationAssertion(terms:contributor obo:CL_4023015 <https://orcid.org/0000-000
 AnnotationAssertion(oboInOwl:hasDbXref obo:CL_4023015 "ILX:0770150")
 AnnotationAssertion(oboInOwl:inSubset obo:CL_4023015 cl:BDS_subset)
 AnnotationAssertion(rdfs:label obo:CL_4023015 "sncg GABAergic cortical interneuron")
-EquivalentClasses(obo:CL_4023015 ObjectIntersectionOf(obo:CL_0000099 ObjectSomeValuesFrom(obo:RO_0002100 obo:UBERON_0000956) ObjectSomeValuesFrom(obo:RO_0002215 obo:GO_0061534) ObjectSomeValuesFrom(obo:RO_0002292 obo:PR_000015325)))
+EquivalentClasses(obo:CL_4023015 ObjectIntersectionOf(obo:CL_4023064 ObjectSomeValuesFrom(obo:RO_0002100 obo:UBERON_0000956) ObjectSomeValuesFrom(obo:RO_0002215 obo:GO_0061534) ObjectSomeValuesFrom(obo:RO_0002292 obo:PR_000015325)))
```
Comment (aleixpuigb) [inline src/ontology/cl-edit.owl:29215]: Changing the parental terms produces that many cells cannot be classified as this cell type anymore.

File: src/ontology/cl-edit.owl [hunk 6]
```diff
@@ -29221,7 +29223,8 @@ AnnotationAssertion(terms:contributor obo:CL_4023016 <https://orcid.org/0000-000
 AnnotationAssertion(oboInOwl:hasDbXref obo:CL_4023016 "ILX:0770151")
 AnnotationAssertion(oboInOwl:inSubset obo:CL_4023016 cl:BDS_subset)
 AnnotationAssertion(rdfs:label obo:CL_4023016 "vip GABAergic cortical interneuron")
-EquivalentClasses(obo:CL_4023016 ObjectIntersectionOf(obo:CL_0000099 ObjectSomeValuesFrom(obo:RO_0002100 obo:UBERON_0000956) ObjectSomeValuesFrom(obo:RO_0002215 obo:GO_0061534) ObjectSomeValuesFrom(obo:RO_0002292 obo:PR_000017299)))
+EquivalentClasses(obo:CL_4023016 ObjectIntersectionOf(obo:CL_4023064 ObjectSomeValuesFrom(obo:RO_0002100 obo:UBERON_0000956) ObjectSomeValuesFrom(obo:RO_0002215 obo:GO_0061534) ObjectSomeValuesFrom(obo:RO_0002292 obo:PR_000017299)))
```
Comment (aleixpuigb) [inline src/ontology/cl-edit.owl:29226]: Changing the parental terms produces that many cells cannot be classified as this cell type anymore.

**Review by aleixpuigb — COMMENTED** [review]
Please see inline comments.

File: src/ontology/cl-edit.owl (hunk 3 above)
Comment (AvolaAmg) [reply src/ontology/cl-edit.owl:29745]: Should be fixed with the next commit

**Comment by aleixpuigb** [comment]
#gogoeditdiff

**Review by aleixpuigb — APPROVED** [review]
Everything looks good now.

**Review by anitacaron — APPROVED** [review]

**Pull request merged**